tests/test_configuration_set.py

```diff
@@ -137,6 +139,41 @@ def test_configurations_load_with_energies_forces():
             )
 
 
+@work_in_tmp_dir()
+def test_configurations_load_with_energies_forces_diff_sizes(
```

Comment:
Great to have this test! Would be good to add a docstring to explain what we're testing here (loading configurations with different molecules)